src/screens/SettingsScreen.js

```diff
@@ -0,0 +1,154 @@
+import PropTypes from 'prop-types';
+import React, { useContext, useState } from 'react';
+import { RefreshControl, SectionList, StyleSheet, TouchableOpacity, View } from 'react-native';
+
+import { NetworkContext } from '../NetworkProvider';
+import { OrientationContext } from '../OrientationProvider';
+import { colors, device, normalize, texts } from '../config';
+import {
+  CardListItem,
+  Icon,
+  RegularText,
+  SafeAreaViewFlex,
+  TextListItem,
+  Title,
+  TitleContainer,
+  TitleShadow,
+  Wrapper
+} from '../components';
+import { SettingsListItem } from '../components/SettingsListItem';
+import { arrowLeft } from '../icons';
+
+export const SettingsScreen = ({ navigation }) => {
+  const { isConnected, isMainserverUp } = useContext(NetworkContext);
+  const { orientation, dimensions } = useContext(OrientationContext);
+  const [refreshing, setRefreshing] = useState(false);
+
+  const keyExtractor = (item, index) => `index${index}-id${item.id}`;
+
+  const renderSectionHeader = ({ section: { title } }) =>
+    !!title && (
+      <View>
+        <TitleContainer>
+          <Title accessibilityLabel={`${title} (Überschrift)`}>{title}</Title>
+        </TitleContainer>
+        {device.platform === 'ios' && <TitleShadow />}
+      </View>
+    );
+
+  const refresh = () => {
+    setRefreshing(true);
+    // TODO: do we need pull to refresh on the settings screen?
+    // we simulate state change of `refreshing` with setting it to `true` first and after
+    // a timeout to `false` again, which will result in a re-rendering of the screen.
+    setTimeout(() => {
+      setRefreshing(false);
+    }, 500);
+  };
```

Comment:
for testing it is okay like this.

if this function stays but is altered, it should be in a `useCallback`

Reply:
see https://github.com/ikuseiGmbH/smart-village-app-app/pull/123#discussion_r530399334